arch/x86/core/multiboot.c

```diff
@@ -11,4 +11,59 @@
 
 struct x86_multiboot_info x86_multiboot_info;
 
+#ifdef CONFIG_X86_MULTIBOOT_FRAMEBUF
+
+#include <display/framebuf.h>
+
+static struct framebuf_dev_data multiboot_framebuf_data = {
+	.width = CONFIG_X86_MULTIBOOT_FRAMEBUF_X,
+	.height = CONFIG_X86_MULTIBOOT_FRAMEBUF_Y
+};
```

Comment:
Not sure I see much value here if the in-memory struct is just populated with hardcoded configuration.  I mean, on a PC who cares, I guess, but in general it's always nicer to have things fall back to immediate constants instead of allocating bytes in .data.

Reply:
This struct is initialized by the device-dependent driver (the x86 multiboot stuff), and is used by the device-independent framebuffer driver. That's why it's separated out.